.github/workflows/master_std_ui.yml

```diff
@@ -150,6 +157,14 @@ jobs:
           name: cypress-videos
           path: cypress/videos
           retention-days: 7
+ 
+      - name: Upload Mochawesome report
+        uses: actions/upload-artifact@v2
+        if: always()
```

Comment:
shouldn't be this also `if: failure()` ?

Reply:
I thought it was needed always as we want also the good results to be displayed, but I may be wrong. 
I will try without it